spark/v4.0/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -177,7 +177,19 @@ public static Object[][] parameters() {
         SparkCatalog.class.getName(),
         ImmutableMap.of("type", "hadoop"),
         FileFormat.PARQUET,
-        RANDOM.nextBoolean(),
+        true,
```

Comment:
I would pull this out into a precursor pr, makes sense though, not sure why we made this random before